llzk-sys/src/sanity_tests/transforms.rs

```diff
@@ -12,56 +12,74 @@ use crate::{
     sanity_tests::{TestContext, context},
 };
 
-#[rstest]
-fn test_mlir_register_transformation_passes_and_create(context: TestContext) {
-    unsafe {
-        mlirRegisterLLZKTransformationPasses();
-        let manager = mlirPassManagerCreate(context.ctx);
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use rstest::fixture;
 
-        let pass1 = mlirCreateLLZKTransformationRedundantOperationEliminationPass();
-        let pass2 = mlirCreateLLZKTransformationRedundantReadAndWriteEliminationPass();
-        let pass3 = mlirCreateLLZKTransformationUnusedDeclarationEliminationPass();
-        mlirPassManagerAddOwnedPass(manager, pass1);
-        mlirPassManagerAddOwnedPass(manager, pass2);
-        mlirPassManagerAddOwnedPass(manager, pass3);
+    #[fixture]
+    fn register_passes() {
+        static ONCE: std::sync::Once = std::sync::Once::new();
+        ONCE.call_once(|| {
+            unsafe {
+                mlirRegisterLLZKTransformationRedundantOperationEliminationPass();
+                mlirRegisterLLZKTransformationRedundantReadAndWriteEliminationPass();
+                mlirRegisterLLZKTransformationUnusedDeclarationEliminationPass();
+            }
+        });
+    }
+
+    #[rstest]
+    fn test_mlir_register_transformation_passes_and_create(register_passes: (), context: TestContext) {
+        unsafe {
+            // mlirRegisterLLZKTransformationPasses();
```

Comment:
These look like commented out code but if it's still here for reference or something, add additional comments explaining why.